### DISCUSSION TITLE
Make theme feel more lighter

### DISCUSSION THREAD

**Discussion by sscherfke** [pradyunsg/furo#586] [Potential Issue]
In October, there was a short [Twitter discussion](https://twitter.com/tintvrtkovic/status/1580578670560784385) about Furo not looking good and were not able to say *why* it did not look. The subject has not left me in pease and maybe I identified few issues:

- Everything looks too compressed.  There could be a bit more whitespace.
- The headings are too strong

I just changed a few parameters in Firefox and imho the result looks a bit more pleasant.  Maybe this could be start for some tweaking?

- `p margin-bottom` from `0.75rem` to `1rem` (give pararaphs more air)
- `h* font-weight` from `700` to `400` (make headings less strong)
- `h2 font-size` from `2em` to `1.75em` (make headings less strong)
- `h* color` from `#ffffffcc` to `#ffffffd9` (compensate for less font weight)
- `h* padding-top` from `0` to `.5rem` (make highlight box for current heading look better)
- `div.content width | padding` from `46em | 0 3em` to `44em | 0 4em` (decrease line lenght of contents, add more whitespace between sidebars and contents)

IMHO, the result looks a bit better.  What do you think?

![2022-11-24_14-48](https://user-images.githubusercontent.com/511179/203804357-be5a61ce-213c-42d1-9d5f-b9707c87b7be.png)
![2022-11-24_15-06](https://user-images.githubusercontent.com/511179/203804361-80adc90d-e89f-4468-82d4-75670514f36e.png)

**Comment by Tinche** [comment]
I can share my changes (from cattrs) a little later. One thing though - that background color on the heading when you get to it via a link feels really off to me - I think it should be much more subtle. Haven't figured out how to change it yet.

**Reply by pradyunsg** [reply]
I see https://github.com/python-attrs/cattrs/blob/main/docs/_static/custom.css which is all of the changes, no?

The background is `var(--color-highlight-on-target)` -- like ~all colours in Furo, it's a user-overridable variable.

**Comment by pradyunsg** [comment]
> * `h* padding-top` from `0` to `.5rem` (make highlight box for current heading look better)

What browser are you on? I don't see this in Firefox, Chrome or Safari (on MacOS).

**Reply by sscherfke** [reply]
Firefox on Linux.

I can apply the same changes to firefox on macos and make new screenshots if you like.

**Reply by pradyunsg** [reply]
That would be helpful, as a separate comment here.

Which Linux distro are you on, and what font is Furo ending up using? I wonder if some of the "erkiness" are based off of a suboptimally-picked font from the system font stack.

**Reply by sscherfke** [reply]
Fedora 35/36.  Firefox uses Nimbus Sans (Chrome would use Liberation Sans).  I double checked that enabled the "default fonts" in firefox.

**Comment by pradyunsg** [comment]
> * `div.content width | padding` from `46em | 0 3em` to `44em | 0 4em` (decrease line lenght of contents, add more whitespace between sidebars and contents)

This is likely worth doing on its own.

**Comment by pradyunsg** [comment]
> * `h* font-weight` from `700` to `400` (make headings less strong)

That's reasonable. This is the third time I've had someone point this out, so I'm on board for reducing this.

> * `h2 font-size` from `2em` to `1.75em` (make headings less strong)

I'm not opposed, but instinctively, I don't like how this changes the following plot -- it's not an argument for or against but pointing out a thing that I noticed. :)

![Screenshot 2022-11-24 at 18 11 56](https://user-images.githubusercontent.com/3275593/203846337-d986b784-3bf8-45a9-aece-e31675d46105.png)

![Screenshot 2022-11-24 at 18 12 14](https://user-images.githubusercontent.com/3275593/203846336-96cfec5f-9e4b-4ac9-84fb-399c34ce006d.png)

> * `h* color` from `#ffffffcc` to `#ffffffd9` (compensate for less font weight)

👍🏽

**Reply by sscherfke** [reply]
I always have problems with finding the right heading sizes, too.  Reducing the h2 font size is not the most important thing on my list, though. :)

**Comment by lognaturel** [comment]
Thanks for this thread, @sscherfke, and for being open to this kind of feedback, @pradyunsg!

I agree that the relative font sizes and weights don't feel right. Reducing the h2 font size is fairly high on my wishlist, as is making the admonition text size the same as the body text. I also feel like the body text size is a touch too big, especially on large monitors.

@pradyunsg are you interested in a proposal for across-the-board changes? If I work with a designer on these (referencing @sscherfke and @Tinche's changes), how much explanation/justification would you like around the changes? I'm asking because if you're not interested in a proposal, we'll make the changes just for us and not worry about justifying them. If you're not sure, we could do something similar to @sscherfke where we share screenshots of our changes and then you could ask for justification where changes feel bad or arbitrary to you.

Thanks for a great starting point and I'd love to be involved in making it even better for everyone if that feels appropriate! 🚀

(I work on ODK, open source tools for mobile data collection mostly used for social impact: https://docs.getodk.org/)

**Comment by lognaturel** [comment]
https://furo-topbar.readthedocs.io/en/latest/ seems to have made a number of the same changes we're talking about in this thread.
